.gitignore

```diff
@@ -24,6 +24,7 @@ CTestTestfile.cmake
 libimagequant
 libimagequant-prefix
 src/deps.mk
+src/entities.json
```

Comment:
listing this in gitignore is good, but then you've actually include the json file below.  you should delete the file from the PR.